x/stake/types/delegation.go

```diff
@@ -24,6 +24,32 @@ type delegationValue struct {
 	Height int64
 }
 
+// DelegationWithoutDec defines a delegation without the type sdk.Dec for shares
+//
+// TODO: Remove this type (ref: #2096)
+type DelegationWithoutDec struct {
+	DelegatorAddr sdk.AccAddress `json:"delegator_addr"`
+	ValidatorAddr sdk.ValAddress `json:"validator_addr"`
+	Shares        string         `json:"shares"`
+	Height        int64          `json:"height"`
+}
+
+// aggregates of all delegations, unbondings and redelegations
+type DelegationSummary struct {
+	Delegations          []DelegationWithoutDec `json:"delegations"`
+	UnbondingDelegations []UnbondingDelegation  `json:"unbonding_delegations"`
+	Redelegations        []Redelegation         `json:"redelegations"`
+}
+
+func NewDelegationWithoutDec(delegation Delegation) DelegationWithoutDec {
```

Comment:
This function shouldn't exist - We should have a function property of `Delegation` which returns what I'll call the `DelegationsForREST` object